doc/source/report/classification.rst

```diff
@@ -76,83 +88,240 @@ selection. The process will also help us to find outliers and missing
-actually indicate different genetic differences. K-fold cross-validation
-might be used to tune model parameters, and a proportion of data would
-be used as test data to evaluate the model performance. Notice that we
-may manually manipulate so that the both the training data and the test
-data cover all strains of mice.
-
-2.Unsupervised clustering In addition, we can use unsupervised machine
-learning models (e.g. K-means) to cluster the daily mouse activities
-into clusters that correspond to the genes. This means we will not use
-the given strain label, instead we will create new labels for mice
-purely based on its behaviors by clustering. This can highlight an even
-stronger relationship between genes and mouse behaviors.
-
-The step after model fitting is to assess the important behavioral
-features in the classification and clustering models. A smaller set of
-feature space containing only top features might be used to gain better
-interpretations of the model.
-
-Testing Framework outline
+**Classification**
+
+For this project, we mainly focus on three classification algorithms, which are random forests, gradient boosting and support vector machines (SVM). 
+
+*Introduction*
+
+Random forests is a notion of the general technique of random decision forests that are an ensemble learning method for classification, regression and other tasks, that operate by constructing a multitude of decision trees at training time and outputting the class that is the mode of the classes (classification) of the individual trees. The method combines Breiman's "bagging" idea and the random selection of features, correcting for decision trees' habit of overfitting to their training set.
+
+Gradient boosting is another machine learning algorithm for classification. It produces a prediction model in the form of an ensemble of weak prediction models, typically decision trees. Gradient boosting fits an additive model in a forward stage-wise manner. In each stage, it introduces a weak learner to compensate the shortcomings of existing weak learners, which allows optimization of an arbitrary differentiable loss function. 
+
+Support vector Machines(SVM) are set of related supervised learning methods for classification and regression, which minimizes the empirical classification error and maximize the geometric margin. SVM map the input vector into a higher dimensional space where the maximal separating hyper plane is constructed. Maximizing the distance between different parallel hyper planes, SVM come up with the classification of the input vector. 
+
+*Tuning Parameters*
+
+For each of the algorithms, we create functions to fit them on the dataset respectively. There are two different ways to fit these methods: if the user pre-defines the set of the parameters, we will use cross validation to find the best estimators and their relative labels; if the user does not define the parameters, the functions will use the default values to fit the models.
```

Comment:
"For each of the algorithms, we create functions to fit them on the dataset. There are two different ways to fit these methods: if the user pre-defines the set of the parameters, we will use cross validation to find the best estimators and their relative labels; if the user does not define the parameters, the functions will use the default values to fit the models."